apps/meteor/server/models/MatrixBridgedRoom.ts

```diff
@@ -0,0 +1,7 @@
+import { registerModel } from '@rocket.chat/models';
+
+import { trashCollection } from '../database/trash';
+import { db } from '../database/utils';
+import { MatrixBridgedRoomRaw } from './raw/MatrixBridgedRoom';
+
+registerModel('IMatrixBridgedRoomModel', new MatrixBridgedRoomRaw(db, trashCollection));
```

Comment:
I guess we don't need to store removed data from this collection right?

```suggestion
registerModel('IMatrixBridgedRoomModel', new MatrixBridgedRoomRaw(db));
```

Reply:
Yes